### PR TITLE
feat(weather): Add weather widget to dashboard

### PR DESCRIPTION
**Pull request by BigGillyStyle**
This PR...

1. Queries the user's location
2. Makes a request to OpenWeatherMap for data based on that location
3. Displays the weather data in a widget in the dashboard with **very basic styling** (but does wrap horizontally and scroll vertically as the container resizes)

Some of the changes in this PR are just due to VS Code applying Prettier to files I've edited.  In a real project I think we should just decide whether or not we integrate Prettier into a pre-commit hook (which I am a fan of)...but for the purpose of this short-lived POC I'm not sure it's worth the time.

### CODE REVIEW

File: src/app/open-weather-map/components/open-weather-map/open-weather-map.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(OpenWeatherMapComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```
Comment (anthonymjones) [inline src/app/open-weather-map/components/open-weather-map/open-weather-map.component.spec.ts]: `it` should be `test` for jest

File: src/app/open-weather-map/services/open-weather-map.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import { OpenWeatherMapService } from './open-weather-map.service';
+
+describe('OpenWeatherMapService', () => {
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [OpenWeatherMapService]
+    });
+  });
+
+  it('should be created', inject([OpenWeatherMapService], (service: OpenWeatherMapService) => {
```
Comment (anthonymjones) [inline src/app/open-weather-map/services/open-weather-map.service.spec.ts]: `test`

File: src/styles.scss [hunk 3]
```diff
@@ -3,3 +3,7 @@
 @import 'app/styles/global';
 
 @import 'app/styles/custom-material-theme';
+
+gridster-item {
```
Comment (anthonymjones) [inline src/styles.scss]: In general, I like to keep the `src/styles.scss` just imports. You can see the import for `app/styles/global`, which would be the place for global styles. Also, I notice we're repeating this in the `app.component.scss`. Is that intentional?

Comment (BigGillyStyle) [reply src/styles.scss]: Nope...just forgot to remove.

**Pull request merged**